docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
 
+## Create your areas of influence
 
-### Discover and enrich data thanks to the CARTO catalog
+In the same way as with the geocoding, you can use a straightforward method for creating your areas of influence using isochrones. Isochrones are lines drawn on a map connecting points at which something occurs or arrives at the same time.
 
-You already know where your company stations are and their activity, now you want to know if they are in optimal locations. You start thinking about which data could be valuable to validate that and decide to check if there can be any correlation with households with no car data. Let's see how CARTOframes can help you finding that data and enrich yours.
+For our analysis, let's create and plot isochrones of 15 minutes by walk. Let's see how to do it:
 
-First, we will use the CARTO [data observatory]() to discover the data we want, data about households with no cars. You can learn more about discovering data in the [data discovery guide]().
 
-```py
-from cartoframes.data.observatory.catalog import Catalog
-from cartoframes.data import enrichment
+```python
+from cartoframes.data.services import Isolines
 
-countries = Catalog().countries()
-usa_datasets = countries.get("usa").datasets()
-acs_demo_tracts = usa_datasets.iloc[84]
-acs_no_car = acs_demo_tracts.variables().iloc[[88]]
+isochrones_df, _ = Isolines().isochrones(starbucks_df, [15*60], mode='walk')
+Map([
+    Layer(isochrones_df),
+    Layer(starbucks_df)]
+)
 ```
 
-Now that we have found the data we were looking for, let's filter out our area of intereset, Arlington. To do that, first we need to load Arlington's boundaries (they can be found [here](https://gisdata-arlgis.opendata.arcgis.com/datasets/census-tract-2010-polygons?geometry=-77.761%2C38.787%2C-76.772%2C38.974)):
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn_iso"
+        src="https://cartoframes.carto.com/kuviz/17291073-7c93-431e-9dad-07e5da307a6f"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
 
-```py
-arlington_tracts = gpd.read_file('./data/tracts/1/Census_Tract_2010_Polygons.shp').to_crs({'init': 'epsg:4326'})
-```
+There they are! You can learn more about them and isodistances in the [location data services guide]().
 
-![Arlington tracts](../img/new-guides/quickstart/arlington_tracts.png)
 
-Let's continue enriching that area with the data we found before. To be able to do so, you have to log in to CARTO. You will need to create an API key and use the method `set_default_credentials` to create a session. If you haven't created an API key yet, check the [authentication guide]() to learn how to get it.
 
-Note: If you don't have an account yet, you can get a [free account]() if you are a student or [get a trial]() if you aren't.
+## Discover and enrich your data thanks to CARTO's Data Observatory
```

Comment:
here i don't know if you need "thanks to CARTO's Data Observatory" i the heading...

Reply:
Agree! I have changed it to "Enrich your data with population data"